sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -601,6 +607,20 @@ abstract class Element<T extends Widget> implements BuildContext {
     return ancestor?.widget;
   }
 
+  Point localToGlobal(Point point) {
```

Comment:
I don't think BuildContext should assume it's using a RenderBox. I'd rather expose the renderObject than add this assumption. (I'm not sure exposing the renderObject is the right solution either though.)